### ISSUE TITLE
Floodlights power drain vastly unreasonably (2000)

### ISSUE THREAD

**Issue by AncientSion** [CleverRaven/Cataclysm-DDA#21852]
Game version: latest experimental

Operating system: win 7

Tiles or curses: tiles

Mods active: yup

### Expected behavior
Vehicle floodlights should be a valid pick if you want to a light an area, for example for the purpose of working on said vehicle at night.

Suggestion: Keep Headlight at 75 drain for its thin light angle.
Make Floodlight drain maybe 500 (and that would already be at the upper limit in my opinion), down from 2000. Note Floodlights have no max angle (i think) and are only blocked by adjacent tiles. For a vehicle with a box shape, you would require 4 floodlights, with each having a 180 degree angle. Considering a vehicle has front headlights, you would in theory require 3 additional floodlights, or more depending on the vehicle layout. With a 75 to 500 drain ratio, one flood light would equal about 6 to 7 headlights, which seems reasonable if you compare the angle of a headlight to the median estimated angle of a floodlight (180, but could be more, though unlikely).

### Actual behavior
Floodlight draws 2000 power, Headlight draws 75 power EDIT: WRONG, it draws 187.
I can literally build a Headlight into every outer tile of my vehicle and hence see about 4 times as many tiles in light while drawing less power than a single Floodlight would do.

### Steps to reproduce the behavior
Try installing Floodlight and Headlight and compare power drain.

**Comment by Coolthulhu** [comment]
Correction: headlight draws 187, not 75.

500 is way too little. 1000 could be considered: at this point floodlight would still provide more light, but it would be mostly just for convenience, since 8 headlights produce roughly comparable amount of light as one floodlight.

**Comment by AncientSion** [comment]
Its really 187. Im not sure where i saw the 75, but i certainly saw it. sorry.

However, this basicly means are on the same page. Im comparing 75 and come up to 500, you base it on 187 and come to about 1000. Its basicly the same ratio.

I would definitly be glad if Floodlights would draw 1000. Anything that makes them useful would be GREAT.
Sorry about the number mix up.

**Comment by Leland** [comment]
Just connecting the dots here for future devs – change to 2000 epower was performed in https://github.com/CleverRaven/Cataclysm-DDA/pull/15744

**Comment by AncientSion** [comment]
Hah, interesting. There was a ton of balance changes, but in regards to Flood Lights the PR basicly reads "Flood Lights power need tenfold", without any reasoning or further explanation. So i suppose it was a shot in the dark in conjunction with all the other changes.
This in my opinion gives merit to the fact that the current power need is just vastly imbalanced and should be buffed considerably (again leading to 1000 / 50 % of current need).

**Comment by kevingranade** [comment]
BTW, this is a duplicate of #20443
A floodlight is literally doing the work of 8x headlights which would require 1498 power.  I have no idea where you're getting 1000 from.
There's a potential rebalance of the entire electrical system we could undertake, but short of that, 1500 or seems like the only viable option.

**Comment by Cosmitz** [comment]
Floodlights are indeed using extremely high power drain, way more than even for the 'conveniance' of not using multiple headlights. Either way, a reduce to 1500 would be entirely fine.

**Comment by AncientSion** [comment]
Well unfortunally the commit is already out, but i still want to comment on @kevingranade 

A floodlight might in theory cover 360 degree, which i think is what you are a basing the 1500 "fair" drain on. However, speaking in practical terms, a floodlight usually is placed on the corner of a vehicle, giving light to at best 2 times 90 degree, which throws off your comparison (in my opinion).

So, for a vehicle, the floodlight usually offers 180 degree light, and even for something like a mobile power / workstation (say 3 tile vehicle with solar panel, storage battery and floodlight to allow welding at night), i would want only 180 degree, at best from my floodlight.
Under current circumstances, this makeshift "working rig" would be better of with a single or a pair of headlights. Which is fine, but a bit weird.

Basicly, in my humble opinion a floodlight should have some advantage over a carlight, and maybe some disadvantage. Right now, as far as i can see, a floodlight is overkill and does make little sense to use.
Instead of plain working with (angle : power need), i would wish there was some clear purpose where a floodlight would be the correct choice as opposed to using a headlight.

Plain said, i think its not a good idea to simply see a floodlight as a 360 degree headlight and scale the power up proportionally.
However, i do appeciate you chiming in, doing a commit this quickly, even if i think its a maybe a bit too quick.

Would it perhaps be an option to give a floodlight a max angle of 180 degree, i.e. have them behave like a (directed) wide-angle headlight and with that being, half their power drain ?

Or, if this is not an option, add something like a "directed floodlight" time that works like ?
I havent played the game as much as other people have, but i feel like a wide-angle, directional light would have some use cases, where as a 360 degree omni directional headlight has not (for my use case so far).
That way, we would have narrow direction lights for vehicles, wide directional lights as a upgrade for vehicles and a survivor crafting time, and floodlights for something like maybe a base...or something ?

**Comment by Cosmitz** [comment]
There is some discussion about the types of lights we want, but for the moment, carlights are pretty code-heavy and not an easily modded item. 'floodlights' in CDDA just serves as a sort of 'easy-on' light solution to headlights.

**Comment by kevingranade** [comment]
If you want a wide angle light with less power consumption than a floodlights, you need to add that.  The floodlight emits light in all directions,  it doesn't matter whether it's usual installation wastes half or more of the emitted light, that is not a valid argument for reducing its power consumption.

**Comment by AncientSion** [comment]
Alright then. Thansk for considering and adding your thoughts @kevingranade
